pkg/controller/metrics.go

```diff
@@ -11,11 +11,21 @@ var (
 		Help: "The amount of memory available in a pool",
 	}, []string{"namespace", "pool"})
 
+	PoolMemoryTotal = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Name: "pool_memory_total",
+		Help: "The total amount of memory of a pool",
+	}, []string{"namespace", "pool"})
+
 	PoolCpusAvailable = prometheus.NewGaugeVec(prometheus.GaugeOpts{
 		Name: "pool_cpus_available",
 		Help: "The amount of cpus available in a pool",
 	}, []string{"namespace", "pool"})
 
+	PoolCpusTotal = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Name: "pool_cpus_available",
```

Comment:
this makes me think its a status and not a total.  maybe?

```suggestion
		Name: "pool_cpus_total",
```

Reply:
_total is correct.  I'll fix that.